Dockerfile

```diff
@@ -1,14 +1,20 @@
-# Environment: ROOT6 on Ubuntu/Bionic:
-FROM ubuntu:bionic
-RUN apt-get -y update && \
-    apt-get -y install \
+FROM centos/python-38-centos7:latest
```

Comment:
Hmm, this base image was last updated a year ago, in June 2021. Weren't there any security updates since then?

Note that for runtime environments, we don't need to use Python 3.8 as for REANA infrastructure components, we can use any Python version. 

Perhaps even build on top of `rootproject/root` which has tags like `6.24.06-centos7` if that would be acceptable.